src/senaite/core/profiles/default/workflows/senaite_sample_workflow/definition.xml

```diff
@@ -788,6 +794,7 @@
     <exit-transition transition_id="rollback_to_receive" />
     <exit-transition transition_id="detach" />
     <exit-transition transition_id="dispatch" />
+    <exit-transition transition_id="listing_multi_results"/>
```

Comment:
I think this transition should not be possible here. Results are entered, submitted and verified already, so there is no point on results entry anymore